### PR TITLE
Create 2018-01-00-Kubernetes-V19-Beta-Windows-Support.md

### PR DESCRIPTION
**Pull request by yixin21**
zh-trans 2018-01-00-Kubernetes-V19-Beta-Windows-Support.md
ref:k8smeetup/website-tasks#3354

### CODE REVIEW

**Comment by yixin21** [comment]
Thanks for your review
/assign @xiangpengzhao

File: master/zh/blog/_posts/2018-01-00-Kubernetes-V19-Beta-Windows-Support.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+---
+title: Kubernetes v1.9 公开 Windows 服务器容器的 Beta 支持
+date: 2018-01-09
+slug: kubernetes-v19-beta-windows-support
+url: /blog/2018/01/Kubernetes-V19-Beta-Windows-Support
+---
+<!--
+---
+title: Kubernetes v1.9 releases beta support for Windows Server Containers
```
Comment (markthink) [inline master/zh/blog/_posts/2018-01-00-Kubernetes-V19-Beta-Windows-Support.md:9]: Kubernetes v1.9 beta 发布版开始支持 Windows 容器

Comment (yixin21) [reply master/zh/blog/_posts/2018-01-00-Kubernetes-V19-Beta-Windows-Support.md:9]: 非常感谢，已修改

Comment (tengqm) [reply master/zh/blog/_posts/2018-01-00-Kubernetes-V19-Beta-Windows-Support.md:9]: Kubernetes 1.9 对 Windows Server 容器提供 Beta 版本支持

**Comment by tengqm** [comment]
/approve cancel

Why this is targeting release-1.16 ?

**Comment by yixin21** [comment]
> /approve cancel
> 
> Why this is targeting release-1.16 ?

sorry, this is targeting the wrong branch

**Pull request closed without merging**